extensions/wikia/Wall/WallMessage.class.php

```diff
@@ -100,6 +135,8 @@ static public function buildNewMessageAndPost( $body, $page, $user, $metaTitle =
 			$userPageTitle = self::addMessageWall( $userPageTitle );
 			$useMasterDB = true;
 		}
+		
+		$someTest = "";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$someTest" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=php%3AS1481)